www/templates/default/html/EventsPromoBar.tpl.php

```diff
@@ -0,0 +1,84 @@
+<?php
+use UNL\UCBCN\Event\Occurrence;
+
+$events = array(
+    'with_image' => array(),
+    'without_image' => array()
+);
+
+foreach($context->getRawObject() as $eventInstance) {
+    if($eventInstance->getImageURL()) {
+        $events['with_image'][] = $eventInstance;
+    }
+    else {
+        $events['without_image'][] = $eventInstance;
+    }
+}
+
+function sortEventsByRecurrenceFlag($a, $b) {
+    $a_rtype = $a->eventdatetime->recurringtype;
+    $b_rtype = $b->eventdatetime->recurringtype;
+
+    if(Occurrence::RECURRING_TYPE_NONE == $a_rtype && Occurrence::RECURRING_TYPE_NONE != $b_rtype) {
+        return 1;
+    }
+    else if(Occurrence::RECURRING_TYPE_NONE == $b_rtype && Occurrence::RECURRING_TYPE_NONE != $a_rtype) {
+        return -1;
+    }
+    else {
+        return 0;
+    }
+}
+
+usort($events['with_image'], "sortEventsByRecurrenceFlag");
+usort($events['without_image'], "sortEventsByRecurrenceFlag");
+$grid_class = '';
+switch (count($events['with_image'])) {
+    case 1:
+        $grid_class = 'wdn-col-one-half';
+    break;
+    case 2:
+        $grid_class = 'wdn-col-one-third';
+    break;
+    default:
+        $grid_class = 'wdn-col-one-fourth';
+    break;
+}
+?>
+<div class="band-results">
+    <div class="section-heading">
+        <h1>Today @ UNL</h1>
```

Comment:
two things about this:

1. should it say something about the current calendar rather than 'UNL'? maybe 'Today @ Calendar Name'
2. there should only be one `<h1>` per page. Because this is above the current `<h1>`, it might make sense to change the heading structure around a bit. We can talk about this

Reply:
This isn't going on every page, just the main calendar, which, it is currently coded to do so, so that's wrong. I can adjust the styles around so this isn't an h1 or the other isn't.